.coveragerc

```diff
@@ -652,7 +652,7 @@ omit =
     homeassistant/components/lookin/light.py
     homeassistant/components/lookin/media_player.py
     homeassistant/components/lookin/sensor.py
-    homeassistant/components/luci/device_tracker.py
+    homeassistant/components/luci/*
```

Comment:
This seems like an unrelated and unneeded change?

Reply:
You are right, I've reverted back changes in `.coveragerc`. Thanks.